_blog/maturation-time.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Paper
+subtitle: Fluorescence maturation time - is it a bug or a feature?
+#status: inactive #active if you want it on the frontpage
+
+description: |
+  Using fluorescent maturation delays to infer gene expression dynamics from static snapshots of cell-to-cell variability.
+
+people: # add peope that are involved in this project
+  - euan
+  - jeremy
+
+layout: blog # do not change this
+image: '/img/posts/Maturation_time/mother_machine_timetrace.jpg'
+#link:
+no-link: false # if you don't want it's own webpage
+last-updated: 2021-8-2
+# do NOT add 0 (such as 06 for June). Will break everything lol
+# take out the underscore in the name.
+---
+
+
+<sub><b>Giants this stands on[^1]:
+M. Elowitz ([lab](https://www.elowitz.caltech.edu/)), P. Cluzel ([lab](https://cluzel.fas.harvard.edu/)), P.-S. Laplace ([wiki](https://en.wikipedia.org/wiki/Pierre-Simon_Laplace)), and D. Prasher ([wiki](https://en.wikipedia.org/wiki/Douglas_Prasher)).</b></sub>  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<sub><b>10 minutes read </b></sub>
+
+
+
+ 
+
+[^1]: and many many more of course! 
+
+
+
+<img src="/img/posts/Maturation_time/FPs.jpg" alt="idp" width="250px" align="right" style="padding:15px;">
+Few methods in the history of science have had a bigger impact than visualizing cellular processes with fluorescent proteins ([Exhibit A: Nobel Prizes 2008](https://www.nobelprize.org/prizes/chemistry/2008/illustrated-information/), [Exhibit B: Nobel Prize 2014](https:/www.nobelprize.org/prizes/chemistry/2014/prize-announcement/), [Exhibit C: This mind-blowing video of a brain](https://www.youtube.com/watch?v=c-NMfp13Uug)).
```

Comment:
I feel like this is a bit sensationalist, I guess it's your thoughts and this blog is about our own view of the research, but I felt like this was a strong statement.

Reply:
I see what you mean. Thanks! I'll give it another round.